java/client/src/main/java/glide/api/commands/ServerManagementClusterCommands.java

```diff
@@ -14,11 +15,10 @@
 public interface ServerManagementClusterCommands {
 
     /**
-     * Get information and statistics about the Redis server. DEFAULT option is assumed. The command
-     * will be routed to all primaries.
+     * Get information and statistics about the Redis server. If no argument is provided, so the
```

Comment:
Why "If"?  this command doesn't allow the user to pass an argument.

Reply:
Thanks you, updated in 10277d3f